client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -98,7 +99,7 @@ export const EquipmentDetailView = () => {
 
   const locations =
     locationData?.nodes.map(location => ({
-      label: location.code,
+      label: formatLocationLabel(location as LocationNode),
```

Comment:
The exact type returned from the hook is not named, and differs slightly from this `location` object. However, it's essentially the same (for this purpose anyway), hence as `as`. If there's a better way that doesn't involve manually defining a type....?

Reply:
I'd just change the `formatLocationLabel` function below to accept `LocationRowFragment` (the type of `location` here)!